conda_smithy/ci_register.py

```diff
@@ -149,7 +162,10 @@ def add_project_to_travis(user, project):
 
         if count > 5:
             print('  * Unable to register the repo on Travis (is it down?)')
-            return
+            print("    Please also ensure that the user/org name is spelled correctly "
+                  "(case sensitiv).")
+
+            sys.exit(1)
```

Comment:
ok, then this needs something different: The problem was that `gegister-ci` calls this methods and then all otheres and does not check if the first works and so the second call to travis related stuff will also fail:

```
        ci_register.add_project_to_travis(owner, repo) # -> doe snot work, but returns ok
        ci_register.travis_token_update_conda_forge_config(args.feedstock_directory, owner, repo) # -> fails as well
```

So I think the right action would be to add a `UnrecoverableException(msg)` and do a 

```
try:
   ... real work
except UnrecoverableException as a:
    print(a)
    return
```

in all cli functions...